src/compressor/QatAccel.cc

```diff
@@ -11,19 +11,27 @@
  * Foundation.  See file COPYING.
  *
  */
-
+#include <qatzip.h>
```

Comment:
@tchaikov the line `#include <qatzip.h>` has been moved from _QatAccel.h_ to _QatAccel.cc_ in this change.